src/assets/styles/_base.scss

```diff
@@ -3,6 +3,20 @@
 *::after {
   box-sizing: border-box;
 }
+:root {
+  --kui-line-height-20: 1.3;
+}
```

Comment:
Whilst this seems to sort out all our problems / need for overwrites. It definitely doesn't feel right setting this here like this. Wondering what to do? For the moment I reused this in our own styles, but alternatively I can drop this "setting a variable we don't own" and move the overwrite on line 15 of _forms.scss up here so its altogether maybe?

Dunno still thinking myself, also would like guidance/ to hear from @adamdehaven when hes around.

Reply:
We don't currently have standardized "base page styles" for native elements (e.g. headers, paragraph tags, etc.).

This is something coming in the future and will be exported from one of our existing packages (e.g. you could import a generic base stylesheet) but still TODO

Reply:
Umm, are these `--kui-line-height-20` vars meant to be set by the end user?

The reason I ask is that the `.k-card-body` below uses --kui-line-height-20` in its styles but as its unset currently and then defaults to something that doesn't look right, so we have to override it.

As soon as I set `--kui-line-height-20` to what it looks like our GUI uses as a standard/default line height, everything falls into place.

Reply:
> The reason I ask is that the `.k-card-body` below uses `--kui-line-height-20` in its styles but as its unset currently and then defaults to something that doesn't look right, so we have to override it.

That is intentional. The CSS custom properties are intended to be the application-level customization layer and the Sass variables are the defaults. This allows Kongponents styles to be customized without resorting to overriding CSS rule sets.

The issue with setting only one `--kui-line-height-XX` property is that it only affects uses of exactly this property. Any property with a number different from `XX` won’t be affected by this setting.

Could you provide us with some before/after screenshots of places where we need to change `--kui-line-height-20`, please?

Reply:
Additionally, I think the base font size should be set on `html` or `body`. Why add `.kuma-ready`?